### PR TITLE
feat: pre-register test1 user to make interaction with boards easier

### PR DESCRIPTION
**Pull request by hthieu1110**
This PR is to resolve this issue #990 created by Manfred. 

<details><summary>Contributors' checklist...</summary>

- [ ] Added new tests, or not needed, or not feasible
- [ ] Provided an example (e.g. screenshot) to aid review or the PR is self-explanatory
- [ ] Updated the official documentation or not needed
- [ ] No breaking changes were made, or a `BREAKING CHANGE: xxx` message was included in the description
- [ ] Added references to related issues and PRs
- [ ] Provided any useful hints for running manual tests
- [ ] Added new benchmarks to [generated graphs](https://gnoland.github.io/benchmarks), if any. More info [here](https://github.com/gnolang/gno/blob/master/.benchmarks/README.md).
</details>

### CODE REVIEW

**Review by moul — APPROVED** [review]

File: examples/gno.land/r/demo/users/z_13_filetest.gno [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+// SEND: 200000000ugnot
+
+import (
+	"strconv"
+
+	"gno.land/r/demo/users"
+)
+
+func main() {
+	{
```
Comment (thehowl) [inline examples/gno.land/r/demo/users/z_13_filetest.gno:12]: why the scope?

File: examples/gno.land/r/demo/users/z_13_filetest.gno [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package main
+
+// SEND: 200000000ugnot
```
Comment (thehowl) [inline examples/gno.land/r/demo/users/z_13_filetest.gno:3]: why the send?

File: examples/gno.land/r/demo/users/z_13_filetest.gno
Comment (thehowl) [inline examples/gno.land/r/demo/users/z_13_filetest.gno:1]: why is this not a normal tests in r/demo/users?

**Comment by hthieu1110** [comment]
thanks @thehowl for your reviews. I moved the test to normal tests, hope that've addressed all your feedbacks.

File: examples/gno.land/r/demo/users/z_13_filetest.gno (hunk 1 above)
Comment (hthieu1110) [reply examples/gno.land/r/demo/users/z_13_filetest.gno:12]: I moved the test to normal test

File: examples/gno.land/r/demo/users/z_13_filetest.gno (hunk 2 above)
Comment (hthieu1110) [reply examples/gno.land/r/demo/users/z_13_filetest.gno:3]: I moved the test to normal test

File: examples/gno.land/r/demo/users/z_13_filetest.gno
Comment (hthieu1110) [reply examples/gno.land/r/demo/users/z_13_filetest.gno:1]: I moved the test to normal test

**Comment by hthieu1110** [comment]
Thanks @thehowl , the file has been removed.

**Comment by thehowl** [comment]
Please remember to allow maintainers to edit the PR.

This allows us to merge in master and make sure that the CI is succeeding. Thanks

**Pull request merged**